erratatool.yml

```diff
@@ -26,3 +26,320 @@ cdn_repos:
 - rhocp-{MAJOR}_DOT_{MINOR}-for-rhel-9-aarch64-rpms
 
 quality_responsibility_name: "OpenShift QE"
+
+boilerplates:
+  image:
+    rhsa:
+      synopsis: OpenShift Container Platform 4.{MINOR}.{PATCH} bug fix and security update
+      # Enable legacy style comment on advisory creation
+      advisory_type_comment: True
+      topic: |
+        Red Hat OpenShift Container Platform release 4.{MINOR}.{PATCH} is now available with updates to packages and images that fix several bugs and add enhancements.
+
+        This release includes a security update for Red Hat OpenShift Container Platform 4.{MINOR}.
+
+        Red Hat Product Security has rated this update as having a security impact of  {IMPACT}. A Common Vulnerability Scoring System (CVSS) base score, which gives a detailed severity rating, is available for each vulnerability from the CVE link(s) in the References section.
+      description: |
+        Red Hat OpenShift Container Platform is Red Hat's cloud computing Kubernetes application platform solution designed for on-premise or private cloud deployments.
+
+        This advisory contains the container images for Red Hat OpenShift Container Platform 4.{MINOR}.{PATCH}. See the following advisory for the RPM packages for this release:
+        
+        https://access.redhat.com/errata/RH[X]A-[YYYY:NNNN]
+        
+        Space precludes documenting all of the container images in this advisory. See the following Release Notes documentation, which will be updated shortly for this release, for details about these changes:
+        
+         https://docs.redhat.com/en/documentation/openshift_container_platform/4.{MINOR}/html/release_notes
+        
+        Security Fix(es):
+        
+        * [component-1: Description of issue and fix #1 (CVE-YYYY-NNNN)]
```

Comment:
these need to be the template var {CVE} ?

Reply:
yes indeed - couldn't come back to the PR yesterday since I was tracking other issues. updating now